### PR TITLE
Feat/#14 header component

### LINKED ISSUE #14: Header Component
Create Header Component
Desktop
Header Component
Mobile
Header mobile

### PR DESCRIPTION
**Pull request by Jaimeandres97**
## Description

Add the header component. Creation of both header components, however one is behind the other, later the transition will be developed with JavaScript

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by Jaimeandres97** [comment]
Looking the code the "hero header" has the display set at none in order to only see the next header.
Hero Header:
hero header
 "Next Header"
header

File: index.html [hunk 1]
```diff
@@ -24,6 +24,23 @@
     />
   </head>
   <body>
+    <!-- Header Component-->
+    <header>
+      <nav class="pre-sticky">
+        <a href="/" class="header__sign-in">Iniciar Sesión</a>
+      </nav>
```
Comment (juancho11gm) [inline index.html]: what's the purpose of the pre-sticky element?

File: index.html [hunk 2]
```diff
@@ -24,6 +24,23 @@
...
+      <nav class="post-sticky">
```
Comment (juancho11gm) [inline index.html]: I think post-sticky is not a conventional name 🤔  maybe **navbar**?

File: index.html (hunk 1 above)
Comment (juancho11gm) [reply index.html]: ok got it, it will be used for the fade-in animation

**Comment by juancho11gm** [comment]
Please solve the conflicts 🤠

File: index.html (hunk 1 above)
Comment (juancho11gm) [reply index.html]: However, I think the styles are not correct
Screen Shot 2022-04-20 at 8 04 20 PM

Comment (Jaimeandres97) [reply index.html]: In the commit 7f402b I wrote a comment on how to set the display property of navbar__hero

File: index.html [hunk 3]
```diff
@@ -25,6 +25,23 @@
...
+        <a href="/" class="header__logo">
+          <img
+            class="header__logo-img"
+            src="./img/star-logo.svg"
+            alt="Star+ logo"
+          />
+        </a>
+        <a href="/" class="header__suscribe">Suscríbete Ahora</a>
+        <a href="/" class="post__sign-in">Iniciar Sesión</a>
+      </nav>
```
Comment (juancho11gm) [inline index.html]:
```suggestion
      <nav class="navbar navbar-hero">
        <a href="/" class="navbar-hero__sign-in">Iniciar Sesión</a>
      </nav>
      <nav class="navbar navbar-content">
        <a href="/" class="navbar-content__logo">
          <img
            class="navbar-content__logo-img"
            src="./img/star-logo.svg"
            alt="Star+ logo"
          />
        </a>
        <a href="/" class="navbar-content__suscribe">Suscríbete Ahora</a>
        <a href="/" class="navbar-content__sign-in">Iniciar Sesión</a>
      </nav>
```

File: index.html (hunk 1 above)
Comment (juancho11gm) [reply index.html]: Thank you

File: css/style.css [hunk 4]
```diff
@@ -24,6 +25,97 @@ body {
...
+  opacity: 1;
+  position: fixed;
+}
+
+/* navbar-hero is behind navbar-content until animation is done */
+
+.navbar-hero {
+  background-color: transparent;
+  padding: 0 8px 0 10px;
+  z-index: 4;
```
Comment (juancho11gm) [inline css/style.css:45]: I think you don't need z-index here. You can play with the HTML layers

File: css/style.css [hunk 5]
```diff
@@ -24,6 +25,97 @@ body {
...
+  max-width: none;
+  padding: 9px 8px;
+  pointer-events: auto;
+  text-decoration: none;
+}
+
+.navbar-content {
+  background-color: var(--primary-color);
+  padding: 0 12px 0 20px;
+  z-index: 4;
```
Comment (juancho11gm) [inline css/style.css:69]: same here

Comment (juancho11gm) [inline css/style.css]:
```suggestion
  z-index: 1;
```

File: css/style.css (hunk 4 above)
Comment (juancho11gm) [inline css/style.css]: it's a good practice to keep z-index as low as possible
```suggestion
  z-index: 1;
```

**Review by juancho11gm — APPROVED** [review]

**Pull request merged**